packages/lit-dev-content/site/docs/components/decorators.md

```diff
@@ -102,14 +102,19 @@ assumptions = {
 };
 
 plugins = [
-  ['@babel/plugin-proposal-decorators', {decoratorsBeforeExport: true}],
+  ['@babel/plugin-proposal-decorators',
+    {
+      version: '2018-09',
+      decoratorsBeforeExport: true
+    },
+  ],
   ["@babel/plugin-proposal-class-properties"],
```

Comment:
Nit: in terms of consistency, this line has double quotes whereas the above has single quotes.

It would depend on whether the user uses json or js config file but maybe we should just use JSON syntax with double quotes for both key and value which will works for both.